--- lib/kicad.ts
+++ lib/kicad.ts
@@ -361,10 +361,10 @@
           (stroke (width 0.254) (type default))
           (fill (type none))
         )
-        (pin passive line (at 3.81 0 180) (length 2.54)
+        (pin passive line (at -3.81 0 0) (length 2.54)
           (name "K" (effects (font (size 1.27 1.27))))
           (number "1" (effects (font (size 1.27 1.27))))
         )
-        (pin passive line (at -3.81 0 0) (length 2.54)
+        (pin passive line (at 3.81 0 180) (length 2.54)
           (name "A" (effects (font (size 1.27 1.27))))
           (number "2" (effects (font (size 1.27 1.27))))
         )
